app/code/Magento/QuoteGraphQl/etc/schema.graphqls

```diff
@@ -6,17 +6,24 @@ type Query {
 }
 
 type Mutation {
+    createEmptyCart: String @resolver(class: "\\Magento\\QuoteGraphQl\\Model\\Resolver\\Cart\\CreateEmptyCart") @doc(description:"Creates empty shopping cart for guest or logged in user")
```

Comment:
Change description to

Creates an empty shopping cart for a guest or logged in user